src/openrct2-ui/windows/EditorObjectiveOptions.cpp

```diff
@@ -362,30 +362,30 @@ static void window_editor_objective_options_set_objective(rct_window* w, int32_t
         case OBJECTIVE_10_ROLLERCOASTERS:
             break;
         case OBJECTIVE_GUESTS_BY:
-            gScenarioObjectiveYear = 3;
-            gScenarioObjectiveNumGuests = 1500;
+            gScenarioObjective.Year = 3;
```

Comment:
I don't feel any of this default behaviours should be in the window. It should be moved to the constructor.

Reply:
I'm not sure about that. They're defaults that the scenario editor finds appropriate. I think the struct should be "neutral" here.